app/scripts/disable-console.js

```diff
@@ -0,0 +1,10 @@
+// Disable console.log in contentscript to prevent SES/lockdown logging to external page
+// eslint-disable-next-line import/unambiguous
+if (
+  typeof process !== 'undefined' &&
+  !process.env.METAMASK_DEBUG &&
```

Comment:
Is this what was intended?
```suggestion
  (!typeof process !== 'undefined' && process.env.METAMASK_DEBUG) &&
```

It seems a little odd to skip this just if `process` is defined - even if not in development.

Reply:
Done in: 31d1dbc